### DISCUSSION TITLE
No SED found , No plot created.

### DISCUSSION THREAD

**Discussion by Delphinenishimwe** [mboquien/cigale#82] [Q&A]
I am currently attempting to fit the spectral energy distribution (SED) of non - AGN galaxy with redshift of  0.56.  I have tried  to modify configuration and attempting various parameters, I'm encountering a persistent error: "No SED found/No plot created." Hoping for your help! Thank you!

The following are the pcigale file and input data: 

[input_data.csv](https://github.com/mboquien/cigale/files/14964083/input_data.csv)
[pcigale.txt](https://github.com/mboquien/cigale/files/14964346/pcigale.txt)

**Comment by mboquien** [comment]
It appears that you are using extremely few models. It is not surprising that the fitting fails. I suggest you increase the sampling on the parameters that make sense in your case. Then CIGALE will do a Cartesian grid. You can find some examples in the [manual](https://gitlab.lam.fr/cigale/manual/-/jobs/artifacts/master/raw/CIGALE_Manual.pdf?job=build).
